### PR TITLE
ARROW-15002: [Python] Fix hypothesis strategy for interval types

### PR DESCRIPTION
**Pull request by notEvil**
see https://github.com/apache/arrow/issues/11744

### CODE REVIEW

**Comment by bkmgit** [comment]
Thanks for your contribution. Issues are tracked in [Jira](https://issues.apache.org/jira). Have created the following issue https://issues.apache.org/jira/browse/ARROW-15002

Can you rename `Fix for trivial issue` to `ARROW-15002: [Python] fix interval types` or something similar

**Comment by bkmgit** [comment]
@cyb70289 Maybe you can take a look?

File: python/pyarrow/tests/strategies.py [hunk 1]
```diff
@@ -105,9 +105,7 @@
     pa.duration,
     st.sampled_from(['s', 'ms', 'us', 'ns'])
 )
-interval_types = st.sampled_from(
-    pa.month_day_nano_interval()
-)
+interval_types = st.just(pa.month_day_nano_interval())
```
Comment (cyb70289) [inline python/pyarrow/tests/strategies.py:108]: Not familiar with Hypothesis. Looks okay to me.
@jorisvandenbossche , will you have a look at this change?

**Review by jorisvandenbossche — APPROVED** [review]
Thanks!

**Comment by cyb70289** [comment]
Thanks @notEvil !

**Pull request closed without merging**